tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/server/package-info.java

```diff
@@ -145,14 +144,14 @@ private Documentation() {
 	 * The example above shows how a default project that only defines one project inside the node_modules folder. Since
 	 * this project is of project type {@code definition}, its package.json file is defined explicitly and overrides the
 	 * one that would have been generated. Note that the reserved string
-	 * {@link org.eclipse.n4js.ide.tests.server.TestWorkspaceManager#SRC SRC} is misleading here: the package.json file
-	 * will be located in the base directory of the project.
+	 * {@link org.eclipse.n4js.ide.tests.server.TestWorkspaceManager#CFG_SRC SRC} is misleading here: the package.json
+	 * file will be located in the base directory of the project.
 	 */
 	static int PACKAGE_JSON;
 
 	/**
 	 * Instead of a module name, also the reserved string
-	 * {@link org.eclipse.n4js.ide.tests.server.TestWorkspaceManager#DEPENDENCIES DEPENDENCIES} can be given. As the
+	 * {@link org.eclipse.n4js.ide.tests.server.TestWorkspaceManager#CFG_DEPENDENCIES DEPENDENCIES} can be given. As the
```

Comment:
Did you check this file? Renamins like this one seem odd since the shown string `DEPENDENCIES` was not renamed.